app/models/public_body/calculated_home_page.rb

```diff
@@ -0,0 +1,52 @@
+# Guess the home page based on the request email domain.
+module PublicBody::CalculatedHomePage
+  extend ActiveSupport::Concern
+
+  included do
+    cattr_accessor :excluded_calculated_home_page_domains, default: %w[
+      aol.com
+      btinternet.com
+      gmail.com
+      gmx.com
+      hotmail.com
```

Comment:
```suggestion
      hotmail.com
      hotmail.co.uk
      live.co.uk
      live.com
```

Add some other variants of the service formerly known as Hotmail - we already have `outlook.com` elsewhere.

WDTK has quite a few bodies with these domains - `316`, `437`, `96`, and `17` respectively.